src/main/java/core/basesyntax/service/impl/DataProcessingServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.StorageFruits;
+import core.basesyntax.model.Activity;
+import core.basesyntax.service.DataProcessingService;
+import java.util.List;
+import java.util.Map;
+
+public class DataProcessingServiceImpl implements DataProcessingService {
+    private Map<String, Integer> result;
```

Comment:
Do you use it somewhere? if yes than better do initialization in constructor, if not i think can remove it.